go/registry/api/runtime.go

```diff
@@ -498,6 +498,22 @@ func (r *Runtime) IsCompute() bool {
 	return r.Kind == KindCompute
 }
 
+// StakingAddress returns the correct staking address for the runtime based
+// on its governance model or nil if there is no staking address under the
+// given governance model.
+func (r *Runtime) StakingAddress() *staking.Address {
+	var acctAddr staking.Address
```

Comment:
Minor: you can get rid of this variable and just return directly from the switch cases

Reply:
Compiler says no:
```
registry/api/runtime.go:508:10: cannot take the address of "github.com/oasisprotocol/oasis-core/go/staking/api".NewAddress(r.EntityID)
```
:D

Reply:
ah, nvm then :D